### PR TITLE
add -f for seeds

### LINKED ISSUE #5990: [CT-1285] [Feature] extend `-f` flag shorthand to other commands
### Is this your first time submitting a feature request?

- [X] I have read the [expectations for open source contributors](https://docs.getdbt.com/docs/contributing/oss-expectations)
- [X] I have searched the existing issues, and I could not find an existing issue for this feature
- [X] I am requesting a straightforward extension of existing dbt functionality, rather than a Big Idea better suited to a discussion

### Describe the feature

PR #5908 added the `-f` shorthand for `--full-refresh`, but I accidentally only allowed it for the `dbt build` command. This shoul dbe extended to `dbt seed` and `dbt run` and ot any other command that takes `--full-refresh`

### Describe alternatives you've considered

NA

### Who will this benefit?

typers

### Are you interested in contributing this feature?

yes

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by dave-connors-3**
resolves #5990 

### Description

Seeds need `-f` too! 

image

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [x] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

**Review by gshank — APPROVED** [review]

**Comment by leahwicz** [comment]
@jtcohen6 would we want this in the 1.3 release?

**Comment by jtcohen6** [comment]
@leahwicz Not worth the hassle / risk of backporting IMO. It will be good to have some exciting user-facing functionality in v1.4! Or we could include in the v1.3.1 patch if needed.

File: .changes/unreleased/Features-20221003-110705.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+kind: Features
+body: extend -f
```
Comment (emmyoop) [inline .changes/unreleased/Features-20221003-110705.yaml:2]: Can you expand this to be a bit more specific on what's being added.  A user should be able to read the changelog message and know generally what's been updated.

```suggestion
body: extend -f flag shorthand for seed command
```

**Review by emmyoop — CHANGES_REQUESTED** [review]

**Review by emmyoop — APPROVED** [review]

**Comment by emmyoop** [comment]
Thanks @dave-connors-3!

**Pull request merged**